File: POC/MauiClientPOC/MauiClientPOC/RestAPIClient/Contracts/UserReturnModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace RestAPIClient.Contracts
{
    public class UserReturnModel
    {
        public string ForeName { get; set; }
        public string SurName { get; set; }
        public string LoginName { get; set; }
        public bool DeviceAccess { get; set; }
        public int Id { get; set; }
        public Session Session { get; set; }
        public int IdleTime { set; get; }
    }
}

File: 4WinsGame/4WinGame/AngularWebFrontend/4WinGame/src/app/Models/UserReturnModel.ts
import { SessionModel } from "./SessionModel";


export class UserReturnModel
{
    constructor(foreName:string, surName:string, loginName: string, deviceAccess:boolean, Id: number, session: SessionModel, idleTime: number){
        this.ForeName = foreName;
        this.SurName = surName;
        this.LoginName = loginName;
        this.DeviceAccess = deviceAccess;
        this.Id = Id;
        this.Session = session;
        this.IdleTime = idleTime;
    }
    ForeName: string;
    SurName: string;
    LoginName: string;
    DeviceAccess: boolean;
    Id: number;
    Session: SessionModel;
    IdleTime: number;
}